src/qforte/quantum_op_pool.h

```diff
@@ -58,6 +55,9 @@ class QuantumOpPool {
     std::string str() const;
 
   private:
+    // TODO: Consider if nocc_ and nvir_ are needed as member variables.
+    // These are needed by fill_pool, but why not have those be fill_pool
+    // function variables?
     /// the number of occupied spatial orbitals
     int nocc_;
```

Comment:
Something to add to the TODO: `nocc_` should be generalized in two ways:
1. Convert to alpha/beta to allow open-shell
2. Generalized to two vectors (so that we can support symmetry later)
3. Generalized to more general cases (e.g. active spaces)

Reply:
Same applies to `nvir_`